diesel_tests/tests/schema.rs

```diff
@@ -60,34 +60,17 @@ impl Comment {
     }
 }
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq, Queryable, Insertable, Associations)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Queryable, Insertable, Associations, Identifiable)]
```

Comment:
Should we follow a particular style for `derive`s (or is this rustfmt's fault...)? There are a few `derive`s in this PR that split lines like this into the following format, which group traits (by origin?). I'm fine with the inconsistency - just checking.
```
#[derive(Debug, Clone, Copy, PartialEq, Eq)]
#[derive(Queryable, Insertable, Associations, Identifiable)]
```